<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselInputDialog</class>
 <widget class="QDialog" name="VesselInputDialog">
  <property name="windowTitle">
   <string>Vessel Tracker â€“ Select Vessels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Table for MMSI / Vessel Name -->
   <item>
    <widget class="QTableWidget" name="tableVessels">
     <property name="columnCount">
      <number>2</number>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>MMSI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <!-- Horizontal layout: Add / Remove buttons -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <!-- OK/Cancel buttons -->
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
